staging/src/k8s.io/pod-security-admission/policy/check_allowPrivilegeEscalation_test.go

```diff
@@ -41,13 +42,23 @@ func TestAllowPrivilegeEscalation(t *testing.T) {
 				}}},
 			expectReason: `allowPrivilegeEscalation != false`,
 			expectDetail: `containers "a", "b", "c" must set securityContext.allowPrivilegeEscalation=false`,
+			allowed:      false,
+		},
+		{
+			name: "windows pod, admit without checking privilegeEscalation",
+			pod: &corev1.Pod{Spec: corev1.PodSpec{
+				OS: &corev1.PodOS{Name: corev1.Windows},
+				Containers: []corev1.Container{
+					{Name: "a"},
+				}}},
+			allowed: true,
```

Comment:
also add a negative test case that an explicitly Linux pod fails if it doesn't set securityContext